parser/parser.go

```diff
@@ -39,7 +39,7 @@ type Import struct {
 type Option struct {
 	Pos lexer.Position
 
-	Name  string  `( "(" @("."? Ident { "." Ident }) ")" | @("."? Ident { "." Ident }) )`
+	Name  string  `( ( "(" @("."? Ident { "." Ident }) ")" | @("."? Ident { "." Ident }) ) "."? )+`
```

Comment:
I suspect we should be capturing more structure here, but it's not clear to me what these things even are. I expect that will fall out once we start generating descriptors, as we'll need the full structure.